docs/get-started/roles-and-permissions/configure_access_roles.md

```diff
@@ -0,0 +1,47 @@
+# Configure access control in PMM
+
+Roles are a vital part of Access control. Roles provide users with access to specific, role-based metrics.
```

Comment:
I don't see anything related to roles in this doc. do we really need it here?

Reply:
probably it should be in the next document

Reply:
Done.